swap_meet/item.py

```diff
@@ -1,2 +1,25 @@
+
 class Item:
-    pass
+    def __init__(self, category = None, condition = 0):
+        self.condition = condition
+        if category:
+            self.category = category
+        else:
+            self.category = ""
```

Comment:
We can turn this into a ternary

```suggestion
    self.category = category if category else ""
```